README.mediawiki

```diff
@@ -99,12 +99,15 @@ You must create these components at modeling time like any other component. Then
     def getElements(self):
         return [x for x in self.device().getMonitoredComponents() if random.random() &gt; 0.314159]</pre>
 
-=== Auto Adjustment of Collection Intervals. ===
+=== Auto Adjustment of the Collection Intervals. ===
 
-By default Zenoss tries to auto adjust collection intervals for Calculated Performance and Datapoint Aggregator datasources to show aligned data intervals on the utilization graphs. It determines collection interval of the target datapoints and uses this interval for the aggregated datasource. If you want to disable this feature, you have uncheck "Use Basis Interval" flare for your aggregated datasources. The "Use Basis Interval" property can be found on the aggregated datasource's configuration menu. When "Use Basis Interval" is checked the Minimum Interval and Maximum Interval properties will be used to set a limit to the basis interval. For instance, when the basis interval equals to 40 seconds and the Minimum Interval value is 60 seconds, the Minimum Interval value will be used as the collection interval, the same rule is applicable to the Maximum Interval property. Minimum interval is 1 second.
+By default, Zenoss tries to auto adjust the collection intervals for both Calculated Performance and Datapoint Aggregator datasources to show aligned data intervals on the utilization graphs. It determines basis collection interval of the target datapoints and uses the interval for the aggregated/calculated datasources. If you want to disable this feature, you have uncheck the "Use Basis Interval" flare for your aggregated/calculated datasources. The "Use Basis Interval" property can be found on the aggregated/calculated datasource's configuration menu. When the "Use Basis Interval" is checked, the Minimum and Maximum Interval properties will be used to set limits of the basis interval. For instance, when the basis interval equals to 40 seconds and the Minimum Interval value is 60 seconds, the Minimum Interval will be used as the collection interval, the same rule is applicable to the Maximum Interval property. Minimum Interval cannot be less than 1 second. Auto adjustment of the collection intervals only works on Zenoss 5.
 
 == Changes ==
 
+;2.4.0
+* The "Collection Intervals" feature was disabled on Zenoss 4.x and refactored. (ZPS-1134)
```

Comment:
Would you also make a corresponding update to README.html? This ZenPack's documentation has been converted already. Alternatively you could make the update to README.html and delete README.mediawiki so this confusion doesn't reoccur.